addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -14,18 +14,36 @@
         <!-- ************************************************************* -->
 
         <record id="fp_nacional" model="account.fiscal.position.template">
+            <field name="sequence">1</field>
             <field name="name">Régimen Nacional</field>
             <field name="chart_template_id" ref="account_chart_template_common"/>
+            <field name="auto_apply" eval="True"/>
+            <field name="vat_required" eval="True"/>
+            <field name="country_id" ref="base.es"/>
+        </record>
+
+        <record id="fp_intra_private" model="account.fiscal.position.template">
+            <field name="sequence">2</field>
+            <field name="name">Régimen Intracomunitario Privado</field>
```

Comment:
Better to call this "EU privado"  In this case, the Intra-community regime is actually not applied.

Reply:
This seems outdated code, and not present anymore in final version, but if you intend to put it, I don't get for what is used.

Reply:
@pedrobaeza If you send in Europe to someone without VAT number, he should get the taxes as it was within Spain (and it should not apply the intra-community)

Reply:
OK, I see... Then I would prefer the naming "Intracomunitario fuera del VIES" and rename the other to "Régimen Intracomunitario (VIES)"-